app/models/host_info_providers/puppet_info.rb

```diff
@@ -61,8 +61,7 @@ def networking_params
 
     def classes_info_hash
       return [] if host.environment.nil?
-      return puppetclass_parameters if Setting[:Parametrized_Classes_in_ENC]
-      host.puppetclasses_names
```

Comment:
You can probably drop this method definition since it's now unused.

Reply:
Good catch!